source/uwp/ObjectModelProjection/ObjectModelProjection.vcxproj

```diff
@@ -196,7 +196,7 @@
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```

Comment:
@beervoley I originally tried changing this to match the WindowsTargetPlatformMinVersion, but the pipeline VMs didn't have that SDK installed. Is 17763 ok? It can't be 10.0 because targeting a newer version makes it incompatible with the CsWinRT projection.